lib/scripterator/runner.rb

```diff
@@ -102,20 +103,25 @@ def run_blocks
     end
 
     def run_loop
-      if @id_list.count > 0
```

Comment:
so will this not run for a set of IDs anymore? Before you could set `ID_LIST` and it would run for those ids (which were not necessarily in a continuous range)